packages/sdk-react/src/components/receivables/InvoiceDetails/CreateReceivable/sections/EntitySection.tsx

```diff
@@ -35,7 +35,24 @@ const detailsGridItemProps = {
   lg: 3,
 };
 
-export const EntitySection = ({ disabled }: ISectionGeneralProps) => {
+/** All available fields for current component */
+const allFields = [
+  'entity_vat_id_id',
+  'tax_id',
+  'fulfillment_date',
+  'purchase_order',
+] as const;
+
+interface IEntitySectionProps extends ISectionGeneralProps {
```

Comment:
Maybe we should follow the MUI style of naming interfaces?
`IEntitySectionProps` => `EntitySectionProps`

Reply:
Hm, why? :)
We always use these options. How is our code style related to MUI code style?

Reply:
I don't recall us adopting any convention, accordingly, we follow the MUI style.

Reply:
I don't see any convention on our side. I personally use this format for all interfaces

Reply:
@MadMed677 Can I personally ask you to avoid such naming and follow the MUI style?

Reply:
I would prefer to keep MUI style in all naming.

Reply:
Updated